afs/media/js/views/components/reports/person.js

```diff
@@ -246,6 +253,75 @@ define([
                         }
                     ]
             });
+            
+            ////// Search Details section //////
+            self.nameSummary = ko.observable();
+            self.groupSummary = ko.observable();
+            self.contactSummary = ko.observable();
+            self.externalURISummary = ko.observable();
+            self.nationalitySummary = ko.observable();
+            self.birthSummary = ko.observable();
+            self.deathSummary = ko.observable();
+
+            const nameData = self.resource()?.name;
+            if (nameData) {
+                self.nameSummary(nameData.map(x => {
+                    const type = self.getNodeValue(x, 'name_type');
+                    const content = self.getNodeValue(x, 'name_content');
+                    return { type, content }
+                }));
+            };
+
+            const groupData = self.resource()?.['member of'];
+            if (groupData) {
+                self.groupSummary([{
+                    group: self.getNodeValue(groupData)
+                }]);
+            }
+
+            const contactData = self.resource()?.['contact point'];
+            if (contactData) {
+                self.contactSummary(contactData.map(x => {
+                    const type = self.getNodeValue(x, 'contact point_type');
+                    const content = self.getNodeValue(x, 'contact point_content');
+                    return { type, content }
+                }));
+            }
+
+            const uriData = self.resource()?.exactmatch
+            if (uriData) {
+                self.externalURISummary(uriData.map(x => {
+                    const content = self.getNodeValue(x)
+                    return { content }
+                }));
+            };
+
+            const nationalityData = self.resource()?.nationality;
+            if (nationalityData) {
+                self.nationalitySummary([{
+                    content: self.getNodeValue(nationalityData)
+                }]);
+            }
+
+            const birthData = self.resource()?.birth?.Birth_time;
```

Comment:
FYI - you can get this info with getNodeValue.  Also, getNodeValue is case insensitive.  So you can do all lowercase names.